### PR TITLE
Sanity - Add Yoga classes image and order field

### PR DESCRIPTION
**Pull request by anthopark**
* YogaClass schema to include `image` and `order` field

### CODE REVIEW

File: sanity-studio/schemas/yoga-class.js [hunk 1]
```diff
@@ -35,5 +43,11 @@ export default {
       type: 'number',
       validation: (Rule) => Rule.min(0),
     },
+    {
+      name: 'order',
+      title: 'Order',
+      type: 'number',
+      hidden: true,
+    },
```
Comment (emilyPhee) [inline sanity-studio/schemas/yoga-class.js:51]: What does `order` stands for here? Is it about order of yoga class cards that the admin can edit?

**Review by emilyPhee — APPROVED** [review]

**Comment by emilyPhee** [comment]
great work @anthopark. I left a question about schemas, thanks.

**Comment by emilyPhee** [comment]
[Sanity: Extend Yoga Class Schema to have a Card image](https://trello.com/c/LgC3R61R/14-sanity-extend-yoga-class-schema-to-have-an-image)

File: sanity-studio/schemas/yoga-class.js (hunk 1 above)
Comment (anthopark) [reply sanity-studio/schemas/yoga-class.js:51]: yes, correct!

**Pull request merged**